astropy/tests/plugins/display.py

```diff
@@ -3,8 +3,6 @@
 This plugin provides customization of the header displayed by pytest for
 reporting purposes.
 """
-from __future__ import (absolute_import, division, print_function,
```

Comment:
wow, not sure how this survived here so long.

Reply:
I had the exact same thought!